### PR TITLE
Bump apptestctl version

### PR DESCRIPTION
**Pull request by fhielpos**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,9 @@ following [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+- Changed
+  - Update `apptestctl` to 0.18.0 for VerticalPodAutoscaler CRD
```
Comment (mcharriere) [inline CHANGELOG.md:9]:
```suggestion
  - Update `apptestctl` to 0.18.0 for VerticalPodAutoscaler and PolicyException CRDs
```

File: Dockerfile [hunk 2]
```diff
@@ -71,6 +71,9 @@ COPY --from=builder ${ATS_DIR}/.venv ${ATS_DIR}/.venv
 
 COPY --from=binaries /binaries/* /usr/local/bin/
 
+# Remove after releasing apptestctl
+COPY --from=quay.io/giantswarm/apptestctl:0.17.0-972c60acb3840ccb876e5b26a7b900e98afa766a /usr/local/bin/apptestctl /usr/local/bin/apptestctl
```
Comment (mcharriere) [inline Dockerfile:75]:
```suggestion
COPY --from=quay.io/giantswarm/apptestctl:0.18.0 /usr/local/bin/apptestctl /usr/local/bin/apptestctl
```

**Review by mcharriere — COMMENTED** [review]
do you think we could push this forward? I just released `apptestctl 0.18.0`

**Comment by mcharriere** [comment]
Dependabot did the trick, so ignore me :) https://github.com/giantswarm/app-test-suite/commit/ff5d1601911df53c5fb1dc54d854c839af99f14f

**Comment by fhielpos** [comment]
Ahhh, great. I had to drop this for a while because of other PSS tasks. Great that someone else actually released apptestctl 😄 , will close this one.

**Pull request closed without merging**